tests/console/yast2_proxy.pm

```diff
@@ -83,19 +83,23 @@ sub run {
         if (check_screen 'yast2_proxy_firewall_enabled') {
             send_key 'alt-p';
             wait_still_screen 3;
+            # now we have a new popup to workaround for question about firewall settings
+            if (check_screen 'yast2_proxy_squid_cannot-open-interface') {
```

Comment:
As you know we want to avoid check_screens with non-zero timeout

Reply:
okay, add 10 sec then